gallery/src/data-visualization/ReactVisDonutChartPage.tsx

```diff
@@ -0,0 +1,180 @@
+      </p>
+
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Data Format Reference</h3>
+        </header>
+
+        <p className="nx-p">
+          Data points for donut charts utilize the <code className="nx-code">RadialChartPoint</code>
+          {' '} interface.
+        </p>
+
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>angle</NxTableCell>
+              <NxTableCell>number</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                This determines the angular size of each wedge.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>color</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>yes</NxTableCell>
+              <NxTableCell>
+                This determines the color of each wedge.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>label</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                The label to show next to the wedge.
+              </NxTableCell>
+            </NxTableRow>
+            <NxTableRow>
+              <NxTableCell>subLabel</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>no</NxTableCell>
+              <NxTableCell>
+                The subLabel to show next to the wedge. This can be used for annotations to the top label.
+              </NxTableCell>
+            </NxTableRow>
+          </NxTableBody>
+        </NxTable>
+      </section>
+      <section className="nx-tile-subsection">
+        <header className="nx-tile-subsection__header">
+          <h3 className="nx-h3">Basic API Reference</h3>
+        </header>
+        <NxTable>
+          <NxTableHead>
+            <NxTableRow>
+              <NxTableCell>Name</NxTableCell>
+              <NxTableCell>Type</NxTableCell>
+              <NxTableCell>Required</NxTableCell>
+              <NxTableCell>Description</NxTableCell>
+            </NxTableRow>
+          </NxTableHead>
+          <NxTableBody>
+            <NxTableRow>
+              <NxTableCell>colorType</NxTableCell>
+              <NxTableCell>string</NxTableCell>
+              <NxTableCell>No</NxTableCell>
+              <NxTableCell>
+                Override to let React-vis know that the colors provided are literal color values
```

Comment:
Needs elaboration

Reply:
> Needs elaboration

I have added a bit of detail to this part. Again, not a whole lot of details on the official docs either. Basically, we need to override react-vis assigning custom colors to the wedges by providing `colorType="literal"`

https://github.com/sonatype/sonatype-react-shared-components/pull/442/files#diff-d3cb9f23dd6b85016a9b3a4521568efea78292d6a2254345b4ce92c0edba9c54R98-R107

Reply:
> I have added a bit of detail to this part. Again, not a whole lot of details on the official docs either. Basically, we need to override react-vis assigning custom colors to the wedges by providing colorType="literal"

What happens if `colorType` is not `literal`?